analyzers/src/SonarAnalyzer.Common/Extensions/ISymbolExtensions.cs

```diff
@@ -37,5 +37,8 @@ internal static class ISymbolExtensions
 
         public static bool IsTopLevelMain(this ISymbol symbol) =>
             symbol is IMethodSymbol { Name: TopLevelStatements.MainMethodImplicitName };
+
+        public static bool IsGlobalNamespace(this ISymbol symbol) =>
+            symbol is INamespaceSymbol { Name: "" };
```

Comment:
```suggestion
            symbol is INamespaceSymbol { Name: string.Empty };
```

Reply:
We cannot use `string.Empty` with pattern matching because it's not a constant.